src/Tools/Altinn.Notifications.Tools.RetryDeadDeliveryReports/README.md

```
@@ -0,0 +1,85 @@
+To protect sensitive information, use .NET User Secrets to store database connection strings and API access keys. This prevents accidentally committing credentials to source control.
+
+#### Setting up User Secrets
+
+1. Navigate to the tool's project directory:
+
+```bash
+cd src/Tools/Altinn.Notifications.Tools.RetryDeadDeliveryReports
+```
+
+2. Initialize user secrets (if not already done):
+```bash	
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 8

(MD010, no-hard-tabs)